Source/Krypton Components/Krypton.Toolkit/Values/FadeValues.cs

```diff
@@ -0,0 +1,108 @@
+﻿#region BSD License
+/*
+ *
+ *  New BSD 3-Clause License (https://github.com/Krypton-Suite/Standard-Toolkit/blob/master/LICENSE)
+ *  Modifications by Peter Wagner(aka Wagnerp) & Simon Coghlan(aka Smurf-IV), et al. 2023 - 2023. All rights reserved.
+ *
+ */
+#endregion
+
+namespace Krypton.Toolkit
+{
+    /// <summary>Controls the values for <see cref="KryptonFormFadeController"/>.</summary>
+    [ToolboxItem(false)]
+    [DesignerCategory(@"code")]
+    public class FadeValues : Storage
+    {
+        #region Static Values
+
+        private const bool DEFAULT_FADING_ENABLED = false;
+
+        private const bool DEFAULT_SHOULD_CLOSE_ON_FADE_OUT = true;
+
+        private const int DEFAULT_FADE_SPEED = 50;
+
+        private const VisualForm? DEFAULT_PARENT_FORM = null;
+
+        const VisualForm? DEFAULT_NEXT_FORM = null;
+
+        #endregion
+
+        #region Public
+
+        /// <summary>Gets or sets a value indicating whether [fading enabled].</summary>
+        /// <value><c>true</c> if [fading enabled]; otherwise, <c>false</c>.</value>
+        [Category(@"Data")]
+        [Description(@"Enables the fading effects. Default is 'false'.")]
+        [DefaultValue(DEFAULT_FADING_ENABLED)]
+        public bool FadingEnabled { get; set; }
```

Comment:
All these designer values need `ShouldSerialise###` and `Reset###` functions to work correctly in designers.

Reply:
Done